elrond-wasm-debug/src/api/big_uint_api_mock.rs

```diff
@@ -91,7 +91,7 @@ binary_operator! {BitXor, bitxor}
 
 fn check_sub_result(result: &BigInt) {
 	if result.sign() == num_bigint::Sign::Minus {
-		panic!(b"Cannot subtract because result would be negative")
+		panic!("Cannot subtract because result would be negative")
```

Comment:
subtract -> substract

Reply:
nope, there is no `s` in English https://dictionary.cambridge.org/dictionary/english/subtract

Reply:
TIL